WordPress/src/main/java/org/wordpress/android/ui/mysite/menu/MenuActivity.kt

```diff
@@ -207,31 +212,39 @@ class MenuActivity : AppCompatActivity() {
         super.onStop()
     }
 
+    @OptIn(ExperimentalMaterial3Api::class)
     @Composable
-    @SuppressLint("UnusedMaterialScaffoldPaddingParameter")
     fun MenuScreen(modifier: Modifier = Modifier) {
-        val scaffoldState = rememberScaffoldState()
-
+        val snackbarHostState = remember { SnackbarHostState() }
         Scaffold(
-            scaffoldState = scaffoldState,
-            snackbarHost = { snackbarHostState ->
-                SnackbarHost(hostState = snackbarHostState)
-            },
+            snackbarHost = { SnackbarHost(snackbarHostState) },
             topBar = {
-                MainTopAppBar(
-                    title = stringResource(id = R.string.my_site_section_screen_title),
-                    navigationIcon = NavigationIcons.BackIcon,
-                    onNavigationIconClick = onBackPressedDispatcher::onBackPressed,
+                TopAppBar(
+                    title = {
+                        Text(text = stringResource(id = R.string.my_site_section_screen_title))
+                    },
+                    navigationIcon = {
+                        IconButton(onClick = { onBackPressedDispatcher::onBackPressed }) {
```

Comment:
Tapping this button does not result in navigating backwards for me. I can only navigate away from the _More_ menu via the hardware back button.

Reply:
Thanks for catching this! I'm puzzled by this because it's calling the exact same function as before. I'll figure this out then let you know when it's resolved.

Reply:
Fixed in [4f3b64c](https://github.com/wordpress-mobile/WordPress-Android/pull/21440/commits/4f3b64c6d6d667f34a0b17149c892c2ddb808567)